internal/extender/overhead.go

```diff
@@ -120,32 +135,24 @@ func (o *OverheadComputer) compute(ctx context.Context) {
 			// pending pod or pod succeeded or failed
 			continue
 		}
-		node, err := o.nodeLister.Get(p.Spec.NodeName)
-		if err != nil {
-			svc1log.FromContext(ctx).Warn("node does not exist in cache", svc1log.SafeParam("nodeName", p.Spec.NodeName))
+
+		// found pod with no associated resource reservation, add to overhead
```

Comment:
nit - make the placement of comments on conditional blocks consistents, right now you have both:

```golang
// found a thing that does foo
if thing == foo {
    continue
}

if thing == bar {
    // found a bar thing
}
```